bowling/player.cpp

```diff
@@ -0,0 +1,78 @@
+#include "player.hpp"
+
+#include <iomanip>
+#include <iostream>
+#include <iterator>
+#include <sstream>
+
+Player::Player() {}
+
+Player::Player(std::string name, std::vector<std::pair<int, int>> points)
+    : name_(name),
+      points_(points) {}
+
+std::string Player::getName() const {
+    return name_;
+}
+
+int Player::getScore() const {
+    return score_;
+}
+
+void Player::setScore(int score) {
+    score_ = score;
+}
+
+void Player::countScore(std::vector<std::pair<int, int>> points) {
+    int result = 0;
+    int lastNormalRound = 9 < points.size() ? 9 : points.size();

```

Comment:
Add a parenthesis for clarity
```suggestion
    int lastNormalRound = (9 < points.size() ? 9 : points.size());
```

Reply:
done